src/main/java/Calculator.java

```diff
@@ -0,0 +1,37 @@
+package main.java;
+import java.util.Scanner;
+public class Calculator {
+    public static void main(String[] args) {
+        int number1, number2, result;
+        char action;
+        Scanner input = new Scanner(System.in);
+        System.out.println("Please enter first number");
+        number1 = input.nextInt();
+        System.out.println("Please provide an action: /, *, -, + ");
+        action = input.next().charAt(0);
+        System.out.println("Please enter second number");
+        number2 = input.nextInt();
+        switch (action) {
+            case '/':
+                result = number1 / number2;
+                System.out.println("The result is:");
```

Comment:
Good practice to add space after semicolon "The result is: "

Reply:
done